src/electionguard/elgamal.py

```diff
@@ -113,10 +113,10 @@ class HashedElGamalCiphertext:
     pad: ElementModP
     """pad or alpha"""
 
-    data: bytes
+    data: str
```

Comment:
These could be `BigInteger` but the truth is they are just hexadecimal strings. `bytes.fromhex` is used instead of the more built in [`hex`](https://docs.python.org/3/library/functions.html#hex) due to some of the unique logic. 

Long term, the better plan would be to remove any of this customization and just use built in python methods.